### PR TITLE
feature individual

### PR DESCRIPTION
**Pull request by jpgonzales27**

### CODE REVIEW

File: src/test/java/salesforce/hooks/IndividualHooks.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    @Before(value = "@GetIndividual or @PostIndividual or @DeleteIndividual or @PatchIndividual ", order = 1)
+    public void generateToken() {
+        log.info("Generate Token Individual");
+        ApiResponse apiResponse = ApiManager.executeToken();
+        tokenUser = apiResponse.getBody(Token.class).getAccess_token();
+        objectInformation.setToken(tokenUser);
+    }
+
+    @Before(value = "@GetIndividual or @DeleteIndividual or @PatchIndividual", order = 2)
+    public void createAccountHooks() throws JsonProcessingException {
```
Comment (GustavoZHA) [inline src/test/java/salesforce/hooks/IndividualHooks.java]: Account change to Individual

File: src/test/java/salesforce/steps/IndividualGeneralSteps.java [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+    @Then("the response status code should be {string} to individual")
+    public void theResponseStatusCodeShouldBeToIndividua(String statusCode) {
+        log.info("I verify status Individual response");
+        Assert.assertEquals(apiResponse.getStatusCode(), HttpStatus.valueOf(statusCode).value());
+        apiResponse.getResponse().then().log().body();
+    }
+
+    @Then("^The schema to individual should be equals to \"([^\"]*)\"$")
+    public void theSchemaToContactShouldBeEqualsTo(String schema) {
```
Comment (GustavoZHA) [inline src/test/java/salesforce/steps/IndividualGeneralSteps.java]: Change Contact to Individual

File: src/test/java/salesforce/steps/IndividualGeneralSteps.java [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+    @Then("^the response status code fail should be (.+) to individual$")
+    public void theResponseStatusCodeFailShouldBeStatusToIndividua(String status) {
+        log.info("I verify status Individual fail response");
+        Assert.assertEquals(apiResponse.getStatusCode(), Integer.parseInt(status));
+        apiResponse.getResponse().then().log().body();
+    }
+
+    @When("^I use endpoint \"([^\"]*)\" request to individual with (.+) and (.+)$")
+    public void iUseEndpointRequestToContactWithApiNameAndValue(String endpoint,String apiName, String value) {
```
Comment (GustavoZHA) [inline src/test/java/salesforce/steps/IndividualGeneralSteps.java]: Here too

File: src/test/java/salesforce/steps/IndividualGeneralSteps.java [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        log.info("I create fail individual");
+        apiRequest.setBody(jsonConvert(apiName,value));
+        apiRequest.setEndpoint(endpoint);
+        apiResponse = ApiManager.execute(apiRequest);
+    }
+
+    @And("^I update individual the (.+) to (.+)$")
+    public void iUpdateContactTheApiNameToValue(String apiname, String value) {
+        log.info("I update the account fail");
```
Comment (GustavoZHA) [inline src/test/java/salesforce/steps/IndividualGeneralSteps.java]: is the individual to update

Comment (jpgonzales27) [reply src/test/java/salesforce/steps/IndividualGeneralSteps.java]: All ready done Gus

**Pull request merged**